sqlmesh/core/scheduler.py

```diff
@@ -140,7 +140,10 @@ def evaluate(
             snapshot.name: snapshot,
         }
 
-        if isinstance(snapshot.model, SeedModel) and not snapshot.model.is_hydrated:
+        if (
+            isinstance(snapshot.node, SeedModel)
+            and not t.cast(SeedModel, snapshot.node).is_hydrated
```

Comment:
Why can't this remain `snapshot.model.is_hydrated`?

Reply:
`is_hydrated` is only supported for `SeedModel` and not part of the general `Model` interface, so it needs to be narrowed down. Maybe `snapshot.node.is_hydrated` will work though. I'll try that.